<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление существа</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_alias">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_alias">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_description">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ability">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Способности:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ability">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_of_stay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Время первого 
появления в 
источниках:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_time_of_stay">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_book">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Источник 
упоминания:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_photo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите картинку: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>441</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_watch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>441</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Посмотреть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_book">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>299</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить книжку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Если в предоставленом списке нет одходящей 
книжки, то вы можете добавить ее: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
